doc/source/tutorials/local/Adding_new_data/Select_write_store.rst

```diff
@@ -0,0 +1,286 @@
+Select a store to write to
```

Comment:
Question: is this file being added in addition to the "Adding observation data" tutorial or were they going to be merged in some way? Separate files sound like a good approach - just wondering about avoiding repetition.